src/App.tsx

```diff
@@ -1,14 +1,43 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { Request } from './components/Request';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const { All, Active, Completed } = Filter;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [selectedFilter, setSelectedFilter] = useState<string>(All);
+  const [query, setQuery] = useState('');
+
+  const todosPoint = 'todos';
+
+  useEffect(() => {
+    Request(todosPoint)
+      .then(data => setTodos(data));
+  }, []);
+
+  const filterTodos = useMemo(() => todos.filter((e) => {
```

Comment:
```suggestion
  const filterTodos = useMemo(() => todos.filter((todo) => {
```
use descriptive variable names

Reply:
fix all such cases